README.md

```diff
@@ -4,9 +4,9 @@ _This repo is the new home of the fall detection model used in Ambianic Edge. We
 
 # TODO
 
-- [ ] Standalone Python ML library for people fall detection based on Tensorflow and [PoseNet 2.0](https://github.com/tensorflow/tfjs-models/tree/master/posenet).
+- [x] Standalone Python ML library for people fall detection based on Tensorflow and [PoseNet 2.0](https://github.com/tensorflow/tfjs-models/tree/master/posenet).
 - [ ] Python source code and wheel package published on PyPi 
-- [ ] Jupyter Notebook to interactively test and experiement with the model
+- [x] Jupyter Notebook to interactively test and experiement with the model
 - [ ] CI & test suite
```

Comment:
@bhavikapanara any reason the CI & test suite checkbox is not checked yet?

Reply:
done